app/scripts/controllers/pod.js

```diff
@@ -316,6 +317,17 @@ angular.module('openshiftConsole')
           }
         };
 
+        var focusTerminal = function() {
+          $('.terminal').focus();
```

Comment:
do we actually have multiple terminals when there are multiple containers (with only one visible)? or do we just maintain the connections?

or does jquery only find the visible ones by default, cant remember